src/lib.rs

```diff
@@ -209,21 +322,89 @@ pub async fn init_redis_connection(
     let manager = RedisConnectionManager::new(format!("redis://{ip_addr}:{port}"))?;
     let pool = Pool::builder().build(manager).await?;
 
```

Comment:
@marioCluml, this line should be 
```
let pool = build_pool_with_options::<RedisConnectionManager>(manager, None).await?;
```
because we want the log-broker users to use our default setting, not bb8's default setting.